internal/provider/resource_cloudflare_worker_script.go

```diff
@@ -129,6 +129,11 @@ func resourceCloudflareWorkerScriptCreate(ctx context.Context, d *schema.Resourc
 		return diag.FromErr(fmt.Errorf("script content cannot be empty"))
 	}
 
+	moduleEnabled := d.Get("module_enabled").(bool)
```

Comment:
I'd recommend `type`: either `"service-worker"` (default) or `"modules"`.

Reply:
I was think something about like `esm-enabled: true` or  just `esm: true`

Reply:
let's match the go library and use `module: true`